src/native/minipal/cpufeatures.c

```diff
@@ -17,8 +17,21 @@
 #include "minipalconfig.h"
 
 #if HAVE_AUXV_HWCAP_H
+
 #include <sys/auxv.h>
 #include <asm/hwcap.h>
+
+// Light-up for hardware capabilities that are not present in older headers used by the portable build.
+#ifndef HWCAP_ASIMDRDM
```

Comment:
We do similar light-up in other places, e.g. here: https://github.com/dotnet/runtime/blob/9cf693a66061b7c1efeeed7053a321fed786d3ac/src/coreclr/pal/src/thread/process.cpp#L67